tsconfig.json

```diff
@@ -6,7 +6,7 @@
     "checkJs": false,
     "jsx": "preserve",
     "declaration": true,
-    "outDir": "./types",
+    "outDir": "./lib",
```

Comment:
Looks like we can remove the `gitignore` for `types` based on this change.

Reply:
Yes. Good catch!